conftest.py

```diff
@@ -120,3 +120,16 @@ def simple_netcdf4(tmpdir):
     ds.to_netcdf(filepath)
 
     return filepath
+
+
+@pytest.fixture
+def random_tiff(tmpdir):
+    from PIL import Image
+
+    array = np.random.randint(0, 255, (128, 128), dtype=np.uint8)
+    img = Image.fromarray(array)
+
+    filepath = tmpdir / "rand.tiff"
+    img.save(filepath)
```

Comment:
The following will save a multi-page TIFF:
```suggestion
    array = np.random.randint(0, 255, (5, 128, 128), dtype=np.uint8)
    imgs = [Image.fromarray(frame) for frame in array]

    filepath = tmpdir / "rand.tiff"
    imgs[0].save(filepath, save_all=True, append_images=imgs[1:])
```